pkg/compliance/spec/compliance.go

```diff
@@ -96,3 +99,16 @@ func securityCheckByCheckID(checkID string) types.SecurityCheck {
 		return types.SecurityCheckUnknown
 	}
 }
+
+// GetComlianceSpec accepct compliance flag name/path and return builtin or file system loaded spec
+func GetComplianceSpec(specNameOrPath string) (string, error) {
+	if strings.HasPrefix(specNameOrPath, "@") {
+		buf, err := os.ReadFile(strings.TrimPrefix(specNameOrPath, "@"))
+		if err != nil {
+			return "", fmt.Errorf("error retrieving compliance spec from path: %w", err)
+		}
+		return string(buf), nil
```

Comment:
What if returning `[]byte` as is? This file content is converted to `string` and then will be converted to `[]byte` just after this function. Seems like it is better to convert the return value of `GetSpecByName` to `[]byte`.